### PR TITLE
Update cluster/README.md

### PR DESCRIPTION
**Pull request by gmarek**
@davidopp

### CODE REVIEW

**Comment by davidopp** [comment]
@erictune can you take a look at this and suggest any changes you think are appropriate? You are more familiar with this directory than me.

**Comment by gmarek** [comment]
Ping @erictune

File: cluster/README.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 The scripts and data in this directory automate creation and configuration of a Kubernetes cluster, including networking, DNS, nodes, and master components.
 
+Tools in this folder are `bash` scripts which make managing single cluster easier, but they are currently not designed to allow easy management of multiple clusters from a single machine. If it's necessary it is possible to replicate existing e2e test cluster setup that consists mainly of `hack/ginkgo-e2e.sh` script and special test configs for specific cloud providers e.g. `cluster/gce/config-test.sh`. This means that in general scripts from `cluster` directory find a running cluster by themselves, and you don't need to explicitly specify the cluster you want to modify, as those tools assume that there's the cluster running.
```
Comment (erictune) [inline cluster/README.md]: Actually, they are supposed to support multiple clusters via the INSTANCE_PREFIX environment variable.  

In terms of what the individual scripts do:
- kube-up.sh makes a new cluster using KUBERNETES_PROVIDER and INSTANCE_PREFIX, etc, and then sets the current-context of your kubeconfig to that cluster
- kube-down.sh deletes the current-context cluster (but you have to set KUBERNETES_PROVIDER and INSTANCE_PREFIX right
- kube-push.sh pushes to the current-context cluster (may also have to set KUBERNETES_PROVIDER and INSTANCE_PREFIX right here)
- kubectl.sh is deprecated.
- I've never had to run any of the other files in cluster/ directly.

Comment (gmarek) [inline cluster/README.md]: I'll just drop this PR.

**Pull request closed without merging**